packages/lib/src/toolbar/controls/Selector/models.ts

```diff
@@ -1,6 +1,3 @@
 import type { ReactElement } from 'react';
 
-export type OptionComponent<T> = (props: {
-  option: T;
-  isSelected?: boolean;
```

Comment:
This prop doesn't seem to be needed anymore (to differentiate an option rendered inside the floating menu or on the trigger once selected).